### DISCUSSION TITLE
find feed to chg feed properties | I have 1000+ feeds

### DISCUSSION THREAD

**Discussion by alpchasm** [lwindolf/liferea#1357] [Q&A]
How do I quickly search for a feed, so that I can change the feed properties -- as in "never update" or change feed description?  My painful method is to export all feeds, then text search and painfully infer what the folder pathname is, then go back into the GUI and navigate to the feed -- takes 10 minutes or so.

The search tool allows me to find articles from the feed, but I see no way to take an article, and work back to adjust  the parent feed / feed properties.

**Comment by lwindolf** [comment]
You are right that is actually hard to do. Can you please open an issue for this?
